packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -328,6 +523,21 @@ private static Bitmap toBitmap(Object o) {
     }
   }
 
+  private static Bitmap toScaledBitmap(Bitmap bitmap, float scale) {
+    if (Math.abs(scale - 1) > 0.001) {
```

Comment:
Can you add a comment on where the magical number `0.001` comes from?

Reply:
and should this method check or document bounds for what values are expected of scale?